src/sql/util.py

```diff
@@ -261,11 +261,15 @@ def get_user_configs(file_path):
 
     Returns
     -------
+    boolean
+        A boolean representing if the user configurations were set successfully.
     dict
         saved configuration settings
     """
     data = load_toml(file_path)
     section_names = ["tool", "jupysql", "SqlMagic"]
+    section_to_find = ""
```

Comment:
I was just thinking that in case there is no value in the `data` variable, the display message checks if `section_to_find` equals "SqlMagic". If it does not enter the while loop, the variable `section_to_find` will not exist and will raise an error.

I did see now that it will always enter the while loop, but I feel it is better in terms of readability and to understand the scope of the variable. If you feel I should remove it, let me know!

Reply:
ok, in that case, it'd be better to initialize to `None`

Reply:
Initialized to `None` [here](https://github.com/ploomber/jupysql/pull/935/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R288)

Reply:
there's something off about the links you're sharing. this is what I see when I click on the link you posted above:

![image](https://github.com/ploomber/jupysql/assets/989250/5573e2c3-d008-4e9e-ab1f-a8dad6c04020)

which doesn't seem relevant to the value returned by the function. am I missing anything?